sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -602,7 +602,8 @@ object DataSource extends Logging {
                   provider1.startsWith("org.apache.spark.sql.hive.orc")) {
                   throw new AnalysisException(
                     "Hive-based ORC data source must be used with Hive support enabled. " +
```

Comment:
Hive built-in ORC data source must be used with Hive support enabled.

Reply:
Yep.